src/SqlUtils.ts

```diff
@@ -5,52 +5,96 @@ import Constants from "./Constants";
 import SqlConnectionConfig from "./SqlConnectionConfig";
 
 export default class SqlUtils {
+
+    /**
+     * Tries connection to server to determine if client IP address is restricted by the firewall.
+     * First tries with master connection, and then with user DB if first one fails.
+     * Returns the client IP address if firewall restriction is present, or an empty string if connection succeeds. Throws otherwise.
```

Comment:
nit: use @returns for doc comment and add @param comment for connectionConfig

Reply:
Fixed